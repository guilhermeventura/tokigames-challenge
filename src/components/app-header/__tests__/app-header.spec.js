/**
 * App Header Test Suite
 *
 * @description Describes the test suite for the AppHeader component
 */

import React from "react";
import { shallow } from "enzyme";
import AppHeader from "../index";

describe("<AppHeader />", () => {
  it("should render without crashing", () => {
    const component = shallow(<AppHeader />);

    expect(component).toMatchSnapshot();
  });
});
