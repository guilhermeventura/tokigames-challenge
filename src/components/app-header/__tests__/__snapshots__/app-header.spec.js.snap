// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppHeader /> should render without crashing 1`] = `ShallowWrapper {}`;
